src/controllers/remove.director.check.answers.controller.ts

```diff
@@ -23,7 +26,9 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
     const session: Session = req.session as Session;
     const companyProfile: CompanyProfile = await getCompanyProfile(companyNumber);
     const companyOfficer: CompanyOfficer = await getDirectorAndTerminationDate(session, transactionId, submissionId);
-    
+    const appointmentId = (await getOfficerFiling(session, transactionId, submissionId)).referenceAppointmentId as string
+    const appointment: CompanyAppointment = await getCompanyAppointmentFullRecord(session, companyNumber, appointmentId)
```

Comment:
should verify that appointment is found. I think getCompanyAppointmentFullRecord returns null on 404.

Reply:
Fixed

Reply:
Turns out the getCompanyAppointmentFullRecord throws errors if not found. Which is different than the other apis so all good.